### DISCUSSION TITLE
change numeric habits relative

### DISCUSSION THREAD

**Discussion by Mr-Tin-25** [iSoron/uhabits#1166] [Feature Requests] [answered]
Hey,

I like to change the numeric value relative.
E.g. If i read 7 pages than I have to set habit number to 10. After a long break I read 25 pages more I have to move to 20. It would be easier to add 10 pages instead of calculating the end result.

Thanks

**Comment by hiqua** [comment, marked as answer]
Is it this: https://github.com/iSoron/uhabits/discussions/951 ?

**Reply by Mr-Tin-25** [reply]
yes
